troubleshooting/src/main/java/RedirectLoggingToFile.java

```diff
@@ -39,8 +40,12 @@
 public final class RedirectLoggingToFile {
 
     public static void main(String[] args) {
-        // Redirect Browser log messages to jxbrowser-browser.log
-        Path loggingDir = Files.createTempDir("jxbrowser-logs");
+        Path loggingDir;
+        try {
+            loggingDir = Files.createTempDirectory("jxbrowser-logs");
+        } catch (IOException e) {
+            throw new RuntimeException("Failed to create temporary directory", e);
+        }
         Path loggingFile = loggingDir.resolve("jxbrowser-browser.log");
```

Comment:
Why `jxbrowser-browser`, btw?

Reply:
Hard to say. I changed to `jxbrowser.log`.

Reply:
It legacy from JxBrowser 6 https://jxbrowser.support.teamdev.com/support/solutions/articles/9000013125-logging